driver-core/src/main/com/mongodb/client/model/TimeSeriesOptions.java

```diff
@@ -104,12 +109,102 @@ public TimeSeriesOptions granularity(@Nullable final TimeSeriesGranularity granu
         return this;
     }
 
+    /**
+     * Returns the maximum time span between measurements in a bucket.
+     *
+     * @param timeUnit the time unit.
+     * @return time span between measurements, or {@code null} if not set.
+     * @since 4.10
+     * @mongodb.server.release 6.3
```

Comment:
Are we sure these options are since 6.3? The unified test specifies `"minServerVersion": "7.0"`.

Reply:
Thank you for bringing up the concern. According to the documentation at [timeseries-procedures](https://www.mongodb.com/docs/v7.0/core/timeseries/timeseries-procedures/) and the server ticket [SERVER-67598](https://jira.mongodb.org/browse/SERVER-67598), this feature has been available since version 6.3. I have created a corresponding pull request to update the spec tests. You can find it at [Pull Request #1433](https://github.com/mongodb/specifications/pull/1433).